File: testprojects/python_main/pipelines/PERF_UNITEST_GENERATE/code/perf_unitest_generate/graph/FlattenSchema_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from perf_unitest_generate.config.ConfigStore import *
from perf_unitest_generate.udfs.UDFs import *

def FlattenSchema_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("account_flags"), col("account_open_date"))

File: testprojects/scala_main/pipelines/SCALA_VECTOR_TYPE/code/src/main/scala/io/prophecy/pipelines/scala_vector_type/graph/FlattenSchema_1.scala
package io.prophecy.pipelines.scala_vector_type.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_vector_type.udfs.UDFs._
import io.prophecy.pipelines.scala_vector_type.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object FlattenSchema_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(col("features"), col("Residential_Land_Zone"))

}